src/main/kotlin/no/nav/familie/ef/sak/fagsak/SøkService.kt

```diff
@@ -101,19 +104,21 @@ class SøkService(
 
     fun søkEtterPersonerMedSammeAdressePåFagsakPerson(fagsakPersonId: UUID): SøkeresultatPerson {
         val aktivIdent = fagsakPersonService.hentAktivIdent(fagsakPersonId)
-        return søkEtterPersonerMedSammeAdresse(aktivIdent)
+        val personopplysninger = personopplysningerService.hentPersonopplysningerFraRegister(aktivIdent)
+        return søkEtterPersonerMedSammeAdresse(aktivIdent, personopplysninger = personopplysninger)
     }
 
     fun søkEtterPersonerMedSammeAdressePåBehandling(behandlingId: UUID): SøkeresultatPerson {
         val (grunnlag) = vurderingService.hentGrunnlagOgMetadata(behandlingId)
 
         val aktivIdent = behandlingService.hentAktivIdent(behandlingId)
-        return søkEtterPersonerMedSammeAdresse(aktivIdent, grunnlag)
+        return søkEtterPersonerMedSammeAdresse(aktivIdent, grunnlag = grunnlag)
     }
 
     private fun søkEtterPersonerMedSammeAdresse(
         aktivIdent: String,
         grunnlag: VilkårGrunnlagDto? = null,
+        personopplysninger: PersonopplysningerDto? = null,
```

Comment:
Kanskje dele denne i to (splitte i en som krever grunnalgsdata og en som krever personopplysninger) og fjerne endepunkt som (kanskje ikke er i bruk)- er det da mulig å sette grunnlag og/eller personopplysninger til not null?

Det vil kanskje bli enklere å bruke denne senere (uten å gjøre feil)